[0.091s] Invoking command in '/home/dyros/ros2_ws/build/dyros_interface': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/dyros/ros2_ws/install/dyros_python:/opt/ros/humble /usr/bin/cmake --build /home/dyros/ros2_ws/build/dyros_interface -- -j2 -l2
[0.486s] -- Found ament_cmake: 1.3.3 (/opt/ros/humble/share/ament_cmake/cmake)
[1.179s] -- Found geometry_msgs: 4.2.2 (/opt/ros/humble/share/geometry_msgs/cmake)
[1.397s] -- Found rosidl_generator_c: 3.1.4 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[1.414s] -- Found rosidl_adapter: 3.1.4 (/opt/ros/humble/share/rosidl_adapter/cmake)
[1.467s] -- Found rosidl_generator_cpp: 3.1.4 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[1.535s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[1.623s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[2.126s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[2.856s] -- Configuring incomplete, errors occurred!
[2.856s] See also "/home/dyros/ros2_ws/build/dyros_interface/CMakeFiles/CMakeOutput.log".
[2.862s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[2.862s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[2.863s]   dyros_interface --arguments-file
[2.863s]   /home/dyros/ros2_ws/build/dyros_interface/rosidl_adapter__arguments__dyros_interface.json
[2.863s]   --output-dir
[2.863s]   /home/dyros/ros2_ws/build/dyros_interface/rosidl_adapter/dyros_interface
[2.864s]   --output-file
[2.864s]   /home/dyros/ros2_ws/build/dyros_interface/rosidl_adapter/dyros_interface.idls)
[2.864s]   returned error code 1:
[2.864s] 
[2.865s]   Error processing 'int a' of 'dyros_interface/AddThreeInts_Request': ''int'
[2.865s]   is an invalid message name.  It should have the pattern
[2.865s]   '^[A-Z][A-Za-z0-9]*$''
[2.865s] 
[2.865s]   Traceback (most recent call last):
[2.866s] 
[2.866s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[2.866s]       return _run_code(code, main_globals, None,
[2.866s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[2.866s]       exec(code, run_globals)
[2.866s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[2.867s]       sys.exit(main())
[2.867s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[2.867s]       abs_idl_file = convert_to_idl(
[2.867s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[2.867s]       return convert_srv_to_idl(
[2.867s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[2.868s]       srv = parse_service_string(package_name, input_file.stem, content)
[2.868s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[2.868s]       request_message = parse_message_string(
[2.868s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 520, in parse_message_string
[2.868s]       Type(type_string, context_package_name=pkg_name),
[2.868s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 277, in __init__
[2.869s]       super(Type, self).__init__(
[2.869s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 205, in __init__
[2.869s]       raise InvalidResourceName(
[2.869s] 
[2.870s]   rosidl_adapter.parser.InvalidResourceName: 'int' is an invalid message
[2.870s]   name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[2.871s] 
[2.871s] Call Stack (most recent call first):
[2.871s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[2.872s]   CMakeLists.txt:12 (rosidl_generate_interfaces)
[2.872s] 
[2.872s] [0m
[2.876s] gmake: *** [Makefile:1179: cmake_check_build_system] Error 1
[2.888s] Invoked command in '/home/dyros/ros2_ws/build/dyros_interface' returned '2': CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:/home/dyros/ros2_ws/install/dyros_python:/opt/ros/humble /usr/bin/cmake --build /home/dyros/ros2_ws/build/dyros_interface -- -j2 -l2
